[2025-02-14T21:33:39.145+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T21:33:39.182+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: group_task_dag.task_group.task7 scheduled__2025-02-13T21:33:00+00:00 [queued]>
[2025-02-14T21:33:39.198+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: group_task_dag.task_group.task7 scheduled__2025-02-13T21:33:00+00:00 [queued]>
[2025-02-14T21:33:39.199+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-14T21:33:39.220+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task_group.task7> on 2025-02-13 21:33:00+00:00
[2025-02-14T21:33:39.225+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=736) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T21:33:39.228+0000] {standard_task_runner.py:72} INFO - Started process 742 to run task
[2025-02-14T21:33:39.234+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'group_task_dag', 'task_group.task7', 'scheduled__2025-02-13T21:33:00+00:00', '--job-id', '839', '--raw', '--subdir', 'DAGS_FOLDER/09_group_task_dag.py', '--cfg-path', '/tmp/tmpbyp4fx4m']
[2025-02-14T21:33:39.236+0000] {standard_task_runner.py:105} INFO - Job 839: Subtask task_group.task7
[2025-02-14T21:33:39.307+0000] {task_command.py:467} INFO - Running <TaskInstance: group_task_dag.task_group.task7 scheduled__2025-02-13T21:33:00+00:00 [running]> on host ab74d4d90d32
[2025-02-14T21:33:39.503+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='group_task_dag' AIRFLOW_CTX_TASK_ID='task_group.task7' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T21:33:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-13T21:33:00+00:00'
[2025-02-14T21:33:39.505+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T21:33:39.542+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-14T21:33:39.543+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 5']
[2025-02-14T21:33:39.557+0000] {subprocess.py:99} INFO - Output:
[2025-02-14T21:33:44.562+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-14T21:33:44.619+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T21:33:44.620+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=group_task_dag, task_id=task_group.task7, run_id=scheduled__2025-02-13T21:33:00+00:00, execution_date=20250213T213300, start_date=20250214T213339, end_date=20250214T213344
[2025-02-14T21:33:44.707+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T21:33:44.758+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T21:33:44.762+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
